characterisation/exceptions/HO200117.test.ts

```diff
@@ -0,0 +1,40 @@
+import World from "../../utils/world"
+import { processPhase2Message } from "../helpers/processMessage"
+import { reasonPath } from "../helpers/errorPaths"
+import generatePhase2Message from "../helpers/generatePhase2Message"
+import MessageType from "../types/MessageType"
+
+jest.setTimeout(30000)
```

Comment:
This can be removed now as we've moved it into https://github.com/ministryofjustice/bichard7-next-tests/blob/main/characterisation/jest.setup.ts.